.github/workflows/main.yml

```diff
@@ -0,0 +1,22 @@
+name: React CI
+on:
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Node.js ${{ matrix.node-version}}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version}}
+        cache: 'npm'
+    - name: Installing dependencies
+      run: |
+        npm install
+        npm ci
```

Comment:
`npm ci` is a special version of `npm install` that is meant to be more efficient to use for continuous integration. You do not need to have both. Just `npm ci` will be enough to install dependencies